src/Metadata/Property/Factory/AnnotationPropertyMetadataFactory.php

```diff
@@ -131,8 +131,11 @@ private function createMetadata(ApiProperty $annotation, PropertyMetadata $paren
 
     private function createWith(PropertyMetadata $propertyMetadata, array $property, $value): PropertyMetadata
     {
-        $getter = $property[0].ucfirst($property[1]);
+        if (null === $value) {
+            return $propertyMetadata;
+        }
```

Comment:
Okay, I'd have kept in from calling this method in the above loop instead of adding the condition here. Weird that we had no issues about this though oO.